### PR TITLE
Updated docs: height, width.

### PR DESCRIPTION
**Pull request by maksimandrianov**
# Issue

What issue is this PR targeting? If there is no issue that addresses the problem, please open a corresponding issue and link it here.

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Update the docs with any new request parameters or changes to behavior described
 - [ ] Update the [changelog](CHANGELOG.md)
 - [ ] If you made changes to the lua files, update the [taginfo](taginfo.json) too.

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: docs/api/turn-by-turn/api-reference.md [hunk 1]
```diff
@@ -115,14 +115,18 @@ These options are available for `auto`, `bus`, and `truck` costing methods.
 | `top_speed` | Top speed the vehicle can go. Also used to avoid roads with higher speeds than this value. `top_speed` must be between 10 and 252 KPH. The default value is 140 KPH. |
 | `ignore_closures` | If set to `true`, ignores all closures, marked due to live traffic closures, during routing. **Note:** This option cannot be set if `location.search_filter.exclude_closures` is also specified in the request and will return an error if it is |
 | `closure_factor` | A factor that penalizes the cost when traversing a closed edge (eg: if `search_filter.exclude_closures` is `false` for origin and/or destination location and the route starts/ends on closed edges). Its value can range from `1.0` - don't penalize closed edges, to `10.0` - apply high cost penalty to closed edges. Default value is `9.0`. **Note:** This factor is applicable only for motorized modes of transport, i.e `auto`, `motorcycle`, `motor_scooter`, `bus`, `truck`, `hov` & `taxi`.
-###### Truck-specific costing options
+###### Other specific costing options
```
Comment (mandeepsandhu) [inline docs/api/turn-by-turn/api-reference.md:118]: s/Other specific costing options/Other costing options

File: docs/api/turn-by-turn/api-reference.md [hunk 2]
```diff
@@ -115,14 +115,18 @@ These options are available for `auto`, `bus`, and `truck` costing methods.
...
+The following options are available for `auto`, `bus`, `how`, `taxi`, and `truck` costing methods.
```
Comment (mandeepsandhu) [inline docs/api/turn-by-turn/api-reference.md:119]: s/how/hov

File: docs/api/turn-by-turn/api-reference.md (hunk 1 above)
Comment (maksimandrianov) [reply docs/api/turn-by-turn/api-reference.md]: Fixed

File: docs/api/turn-by-turn/api-reference.md (hunk 2 above)
Comment (maksimandrianov) [reply docs/api/turn-by-turn/api-reference.md]: Thanks! Fixed

**Review by genadz — APPROVED** [review]

**Pull request merged**